Tensile/TensileCreateLibrary.py

```diff
@@ -682,6 +684,64 @@ def noBuildError(kernel):
     return list(filter(noBuildError, kernels))
 
 
+def getKernelSourceAndHeaderCode(ko: KernelWriterBase) -> Tuple[int, List[str], List[str], str]:
+    """Get the source and header content for a kernel object.
+
+    Arguments:
+        ko: Kernel object to extract content from.
+
+    Returns:
+        Tuple of data: (error code, source code, header code, kernel name)
+
+    Notes:
+        - This function is very similar to **processKernelSource**---the functionality
+        should be merged.
```

Comment:
Do we want this in the docs?

Reply:
If we don't add it somewhere in the docs, I suspect it will never be resolved. Alternatively, we could track it with a ticket.

Reply:
I would prefer the latter